composer.json

```
@@ -41,39 +42,41 @@
         "elasticsearch/elasticsearch": "^7.17||^8.5",
         "ezyang/htmlpurifier": "^4.14",
         "guzzlehttp/guzzle": "^7.4",
-        "laminas/laminas-captcha": "^2.12",
-        "laminas/laminas-code": "^4.5",
-        "laminas/laminas-db": "^2.15",
-        "laminas/laminas-di": "^3.7",
+        "laminas/laminas-captcha": "^2.16",
+        "laminas/laminas-code": "^4.8",
+        "laminas/laminas-db": "^2.17",
+        "laminas/laminas-di": "^3.12",
         "laminas/laminas-escaper": "^2.10",
         "laminas/laminas-eventmanager": "^3.5",
         "laminas/laminas-feed": "^2.17",
-        "laminas/laminas-file": "^2.11",
+        "laminas/laminas-file": "^2.12",
         "laminas/laminas-filter": "^2.17",
         "laminas/laminas-http": "^2.15",
         "laminas/laminas-i18n": "^2.17",
         "laminas/laminas-mail": "^2.16",
         "laminas/laminas-mime": "^2.9",
         "laminas/laminas-modulemanager": "^2.11",
         "laminas/laminas-mvc": "^3.3",
-        "laminas/laminas-oauth": "^2.4",
+        "laminas/laminas-oauth": "^2.5",
```

What's not compatible with version `2.4` in this pull request?
```suggestion
        "laminas/laminas-oauth": "^2.4",
```